scripts/dev_shell.sh

```diff
@@ -16,4 +16,4 @@ source "$SCRIPT_DIR"/constants.sh
 # Start a dev shell with the avalanchego flake
 FLAKE="github:ava-labs/avalanchego?ref=${AVALANCHE_VERSION}"
 echo "Starting nix shell for ${FLAKE}"
-nix develop "${FLAKE}" "${@}"
+nix develop "${FLAKE}"
```

Comment:
```suggestion
nix develop "${FLAKE}" "${@}"
```

Optional to revert this change. This is needed to start the dev shell on my mac because MacOS treats "${@}" as an unset variable when no arguments are passed in, whereas other platforms treat it as an empty string. Since we use `set -euo pipefail` in all of our bash scripts, this causes the script to fail.